lib/rubocop/cop/chef/style/comments_format.rb

```diff
@@ -66,11 +66,11 @@ def on_new_investigation
 
           private
 
+          #
+          # @return [Boolean]
+          #
           def invalid_comment?(comment)
-            comment_types = %w(Author Cookbook Library Attribute Copyright Recipe Resource Definition License)
-            comment_types.any? do |comment_type|
-              /^#\s*#{comment_type}\s+/.match(comment.text)
-            end
+            /^#\s*(Author|Cookbook|Library|Attribute|Copyright|Recipe|Resource|Definition|License)\s+/.match?(comment.text)
```

Comment:
Since this method is called in a loop, you could probably move this regex to a constant and prevent some allocations.

Reply:
Better?

Reply:
Yeah, looks good 👍